### ISSUE TITLE
Astro v5 support?

### ISSUE THREAD

**Issue by beholdr** [vernak2539/astro-rehype-relative-markdown-links#66]
Hi!
Peer deps not allowing Astro v5. Could you please check if it works with v5?

**Comment by vernak2539** [comment]
Hey @beholdr! Thanks for the issue submission!

I didn't even realise that v5 of Astro was out. Thanks for letting me know! 

I've had a quick go at upgrading my own blog to use Astro v5, and `astro-rehype-relative-markdown-links` seems to work just fine. 

I do get this message after upgrading to Astro v5 (I use yarn), but this shouldn't be a blocker for any package managers/projects as it's just a warning.

```
➤ YN0060: │ astro is listed by your project with version 5.0.2, which doesn't satisfy what astro-rehype-relative-markdown-links (pd3e4d) requests (>=2.0.0 <5.0.0-0).
```

That being said, I'm in the process of getting a new release out soon. I'll make sure to test a bit more and hopefully get the peer dep version changes into the release.

**Comment by vernak2539** [comment]
Released in [v0.17.0](https://www.npmjs.com/package/astro-rehype-relative-markdown-links/v/0.17.0) 🎉

**Comment by beholdr** [comment]
Thank you!
